spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -107,6 +119,26 @@ def expect_retryable_error(error_class)
       end
     end
 
+    context 'with non-MAS-related diagnostic code' do
+      let(:submission) do
+        create(:form526_submission,
+               :with_uploads,
+               user_uuid: user.uuid,
+               auth_headers_json: auth_headers.to_json,
+               saved_claim_id: saved_claim.id)
+      end
+
+      it 'does not set a classification code for irrelevant claims' do
+        VCR.use_cassette('evss/disability_compensation_form/submit_form_v2') do
+          subject.perform_async(submission.id)
+          described_class.drain
+          mas_submission = Form526Submission.find(Form526JobStatus.last.form526_submission_id)
+          expect(mas_submission.form.dig('form526', 'form526',
+                                         'disabilities').first['classificationCode']).to eq 'string'
```

Comment:
Why do we expect 'string'?

Reply:
If you can change the factory to produce something more realistic, that would be great! If that change causes too many hard-to-fix failures, then I can approve this PR.